addon/server.js

```diff
@@ -125,26 +125,37 @@ export default class Server {
     });
   }
 
-  create(type, overrides) {
-    var collection = this.schema ? pluralize(camelize(type)) : pluralize(type);
-    var currentRecords = this.db[collection];
-    var sequence = currentRecords ? currentRecords.length: 0;
+  create(type, overrides, collectionFromCreateList) {
+
+    // Store sequence for factory type as instance variable
+    this.factorySequences = this.factorySequences || {};
+    this.factorySequences[type] = this.factorySequences[type] || 0;
```

Comment:
Forgot current implementation was 0-based. Let's just make this line

```js
this.factorySequences[type] = this.factorySequences[type] + 1 || 0;
```

then we can drop line 133, and drop the `- 1` from line 143. I think that's a bit easier to follow.